packages/reacord/library/core/reacord.tsx

```diff
@@ -56,15 +62,18 @@ export abstract class Reacord {
       // eslint-disable-next-line unicorn/no-null
       null,
       "reacord",
-      () => {},
+      () => { },
       // eslint-disable-next-line unicorn/no-null
       null,
     )
 
+
     const instance: ReacordInstance = {
       render: (content: ReactNode) => {
         reconciler.updateContainer(
-          <InstanceProvider value={instance}>{content}</InstanceProvider>,
+          <InstanceProvider value={instance}>
+            {this.wrapper ? this.wrapper({ renderer, initialContent, children: content }) : content}
```

Comment:
This is a bit of an ugly hack that could be implemented better, it essentially gives you the equivalent of an _app.tsx from NextJS where you're able to wrap all of your components with something custom

Reply:
I don't think the utility of this is worth making the API more complex; you can make functions and reuse them too